File: week7/Overloading.cpp
#include<iostream>
using namespace std;
class Venkat
{
    public:
        void variant()
        {
            cout<<"I have a variant where we are similar in few ways\n";
        }
        void variant(string variantName)
        {
            cout<<"VariantName : "<<variantName<<endl;
        }
};
int main()
{
    Venkat obj;
    obj.variant(),obj.variant("A.SaiVenkat");
    return 0;
}

File: week7/Overloading.java
class Venkat
{
    public
        void variant()
        {
            System.out.println("I have a variant where we are similar in few ways");
        }
        void variant(String variantName)
        {
            System.out.println("VariantName : "+variantName);
        }
}
public class Overloading 
{
    public static void main(String[] args) 
    {
        Venkat obj=new Venkat();
        obj.variant();
        obj.variant("A.SaiVenkat");
    }
}
